gym_electric_motor/state_action_processors/flux_observer.py

```diff
@@ -0,0 +1,97 @@
+import gym
+import numpy as np
+
+import gym_electric_motor as gem
+from gym_electric_motor.state_action_processors import StateActionProcessor
+
+
+class FluxObserver(StateActionProcessor):
+    """The FluxObserver extends the systems state vector by estimated flux states ``psi_abs``, and ``psi_angle``.
```

Comment:
I think it would be useful to mention that this processor is to be used for induction machines.